src/adagrams.js

```diff
@@ -1,15 +1,216 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+class Adagrams {
+  constructor() {
+    this.letterPool = [
+      "A",
+      "A",
+      "A",
+      "A",
+      "A",
+      "A",
+      "A",
+      "A",
+      "A",
+      "B",
+      "B",
+      "C",
+      "C",
+      "D",
+      "D",
+      "D",
+      "D",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "E",
+      "F",
+      "F",
+      "G",
+      "G",
+      "G",
+      "H",
+      "H",
+      "I",
+      "I",
+      "I",
+      "I",
+      "I",
+      "I",
+      "I",
+      "I",
+      "I",
+      "J",
+      "K",
+      "L",
+      "L",
+      "L",
+      "L",
+      "M",
+      "M",
+      "N",
+      "N",
+      "N",
+      "N",
+      "N",
+      "N",
+      "O",
+      "O",
+      "O",
+      "O",
+      "O",
+      "O",
+      "O",
+      "O",
+      "P",
+      "P",
+      "Q",
+      "R",
+      "R",
+      "R",
+      "R",
+      "R",
+      "R",
+      "S",
+      "S",
+      "S",
+      "S",
+      "T",
+      "T",
+      "T",
+      "T",
+      "T",
+      "T",
+      "U",
+      "U",
+      "U",
+      "U",
+      "V",
+      "V",
+      "W",
+      "W",
+      "X",
+      "Y",
+      "Y",
+      "Z",
+    ];
+  }
+  drawLetters = () => {
+    // Implement this method for wave 1
+    // create a copy of letter pool so we don't alter original array
```

Comment:
Calling this out with a comment is a good call.

Not copying the array can lead to some really weird bugs!